<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="btnSend">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>103</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>读取</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRemotePort">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>94</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearRecv_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>清空接收</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearRecv">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>接收区</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConnect">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>120</y>
     <width>103</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRemoteIP">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>40</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>21</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>服务器 IP</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtRecv">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>514</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>66</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>端口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxRegType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>380</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>DM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>R</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>B</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MR</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>寄存器类型</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCount">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>360</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxFormatType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>.U</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.D</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>.H</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>软元件数据格式(后缀)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMorDStart">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>430</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>410</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>起始软元件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtValueWrite">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>500</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>00 01 03</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>480</y>
     <width>291</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>写入数据格式以空格键间隔(如:1 3 22)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWrite">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>480</y>
     <width>103</width>
     <height>39</height>
    </rect>
   </property>
   <property name="text">
    <string>写入</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>171</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>接收</string>
   </property>
   <widget class="QCheckBox" name="ckRecvHex">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>17</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>十六进制显示</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckTimestamp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>时间戳</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnWrite</sender>
   <signal>clicked()</signal>
   <receiver>Widget</receiver>
   <slot>on_btnSend_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>614</x>
     <y>706</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>679</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_btnSend_clicked()</slot>
 </slots>
</ui>
